libs/designsystem/menu/src/menu.component.ts

```diff
@@ -271,14 +282,33 @@ export class MenuComponent implements AfterViewInit, AfterContentInit, OnDestroy
   ngAfterContentInit(): void {
     this.setupAccessibilityForItems();
     this.setupAccesibilityForUserProvidedButton();
+    this.setSelectableOnItems();
+  }
+
+  setSelectableOnItems() {
```

Comment:
Alternative name could be:
```suggestion
  ensureSelectableItems() {
```